include/camp/defines.hpp

```diff
@@ -104,6 +104,13 @@ namespace camp
 #endif
 #endif
 
+// libstdc++ from GCC below version 5 lacks the type trait
+#if defined(__GLIBCXX__) && __GLIBCXX__ < 20150422
```

Comment:
Turns out the date-based release numbers for libstdc++ overlap between major versions, we need a release after that date with at least major version 5.

```suggestion
#if defined(__GLIBCXX__) && (__GLIBCXX__ < 20150422 || __GNUC__ < 5)
```